src/_locales/ru/messages.json

```diff
@@ -91,45 +91,45 @@
     "html_options_suspend_screen_capturing_screen_only": { "message": "Захватывать только верх экрана" },
     "html_options_suspend_screen_capturing_entire_page": { "message": "Захватывать всю страницу" },
     "html_options_suspend_screen_capturing_tooltip_line1a": {"message": "На заблокированной вкладке расширение может отображать предварительный просмотр изображения вкладки"},
-    "html_options_suspend_screen_capturing_tooltip_line1b": {"message": " как раз перед тем, как она была приостановлена." },
-    "html_options_suspend_screen_capturing_tooltip_line2a": {"message": "Функция захвата экрана является экспериментальной и может привести к значительным нагрузке на процессор"},
+    "html_options_suspend_screen_capturing_tooltip_line1b": {"message": "как раз перед тем, как она была приостановлена." },
+    "html_options_suspend_screen_capturing_tooltip_line2a": {"message": "Функция захвата экрана является экспериментальной и может привести к значительной нагрузке на процессор,"},
     "html_options_suspend_screen_capturing_tooltip_line2b": {"message": "а также повышенное использование памяти." },
-    "html_options_suspend_screen_capturing_tooltip_line3a": { "message": "Если вы заметили странное поведение, например долгая приоставновка вкладки, или Chrome" },
-    "html_options_suspend_screen_capturing_tooltip_line3b": { "message": "неожиданно падает, попробуйте отключить эту функцию." },
+    "html_options_suspend_screen_capturing_tooltip_line3a": { "message": "Если вы заметили странное поведение, например долгая приоставновка вкладки или" },
+    "html_options_suspend_screen_capturing_tooltip_line3b": { "message": "внезапные вылеты Chrome, попробуйте отключить эту функцию." },
     "html_options_suspend_force_screen_capture": { "message": "Включить режим высококачественного захвата экрана" },
     "html_options_suspend_force_screen_capture_tooltip_line1": { "message": "Когда отключено, снимки экрана имеют следующие ограничения для обеспечения производительности:" },
     "html_options_suspend_force_screen_capture_tooltip_line2": {"message": "- Низкое качество захвата экрана"},
     "html_options_suspend_force_screen_capture_tooltip_line4": {"message": "- Не будет отображаться экран, если для генерации изображения требуется более 1 минуты"},
     "html_options_suspend_force_screen_capture_tooltip_line5": {"message": "- Максимальная высота захвата экрана ограничена 5000px"},
     "html_options_suspend_force_screen_capture_tooltip_line6": {"message": "Включение режима высококачественного захвата экрана может значительно увеличить загрузку процессора"},
-    "html_options_suspend_force_screen_capture_tooltip_line7": {"message": " и использование памяти." },
+    "html_options_suspend_force_screen_capture_tooltip_line7": {"message": "и использование памяти." },
```

Comment:
Also not sure bout deleted space..

Reply:
old version

Reply:
Saw ur final variant, it's better, no changes needed in my view 👍

Reply:
ok